src/script/main/app.ts

```diff
@@ -384,12 +385,14 @@ export class App {
       const serviceMiddleware = new ServiceMiddleware(conversationRepository, userRepository, selfUser);
       const quotedMessageMiddleware = new QuotedMessageMiddleware(this.service.event);
       const readReceiptMiddleware = new ReceiptsMiddleware(this.service.event, conversationRepository, selfUser);
+      const repliesUpdaterMiddleware = new RepliesUpdaterMiddleware(this.service.event);
 
       eventRepository.setEventProcessMiddlewares([
         serviceMiddleware,
         readReceiptMiddleware,
-        eventStorageMiddleware,
         quotedMessageMiddleware,
+        eventStorageMiddleware,
+        repliesUpdaterMiddleware,
```

Comment:
But updating the replies to a message needs to happen once the original message is saved in DB